tests/testthat/test-rd-inherit.R

```diff
@@ -325,17 +325,35 @@ test_that("multiple @inheritParam inherits from existing topics", {
 })
 
 
-test_that("@inheritParam can cope with multivariable argument definitions", {
+test_that("@inheritParam can inherit multivariable arguments", {
   out <- roc_proc_text(rd_roclet(), "
-                       #' My merge
-                       #'
-                       #' @inheritParams base::merge
-                       mymerge <- function(x, y) {}")[[1]]
-  params <- out$get_value("param")
-  expect_equal(length(params), 2)
-  expect_equal(sort(names(params)), c("x", "y"))
+    #' A
+    #' @param x,y X and Y
+    A <- function(x, y) {}
+
+    #' B
+    #'
+    #' @inheritParams A
+    B <- function(x, y) {}"
```

Comment:
Should you add a test where the args here are `.x` and `.y` to ensure that works? I tested it and it does work right, but a test may keep it from breaking in the future

Reply:
Good catches, thanks